### DISCUSSION TITLE
Can I get game ID(s) from command line?

### DISCUSSION THREAD

**Discussion by alterNERDtive** [Heroic-Games-Launcher/HeroicGamesLauncher#3554] [Q&A]
Is there a way to get game IDs from command line, à la `lutris -l`? If not, is there a one liner or simple script that can do it So far my search has come up empty.

After the fact I want to be able to pass them back to `heroic heroic://launch/legendary/${id}`.

I know I can get them from Desktop shortcuts or adding games to Steam, but I’d rather have something that involves less clicking around.

**Comment by imLinguin** [comment]
The ID is also available in the game settings at the bottom. At the moment we don't have any CLI for this.
![image](https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/assets/62100117/9a3ffb55-dcf7-4dae-beca-7f9b2a7fabaf)

**Reply by alterNERDtive** [reply]
Thanks, I guess I missed that. That’s definitely less involved than my approaches :)
